osu.Game/Rulesets/Scoring/HitDetail.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Scoring
+{
+    public enum HitDetail
```

Comment:
I don’t think this really warrants its own enum, it could just be done directly from the `TimeOffset`.

Either way, `HitResultAccuracy` is a better name, “Detail” is very arbitrary.

Reply:
How about `HitTiming` or `HitOffset`?